fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/WindowFunctionChecker.java

```diff
@@ -315,6 +316,9 @@ public Lead visitLead(Lead lead, Void ctx) {
      */
     @Override
     public FirstOrLastValue visitFirstValue(FirstValue firstValue, Void ctx) {
+        if (firstValue.arity() == 2 && firstValue.child(1).equals(BooleanLiteral.TRUE)) {
```

Comment:
does the second child could be any expression or only could be boolean literal? not see any check in `FirstOrLastValue`. Should we remove `visitFirstValue` directly?

Reply:
The second parameter can be a constant expression, but it will be constant-folded. When visitFirstValue has no second parameter or the second parameter is false, there should be no problem and it can be left unchanged.